testnet/lighthouse/validator.toml

```diff
@@ -4,4 +4,8 @@ secrets-dir = "./testnet-keystore/validators"
 beacon-nodes = "http://127.0.0.1:4000"
 init-slashing-protection = true
 http = true
-http-port = 5062
\ No newline at end of file
+http-port = 5062
+metrics = true
+metrics-address="127.0.0.1"
+metrics-allow-origin="*"
```

Comment:
I think we can remove it (previous comment explained it).

Reply:
Done